### PR TITLE
Nuxt3 implement po c for bootstrap vue next on nuxt3 branch css fix

### PR DESCRIPTION
**Pull request by LucasSaid005**
### 📚 Description

_Describe your changes in detail. Why is this change required? What problem does it solve?_

----

### ❓ Type of change

_What types of changes does your code introduce? Put an `x` in all the boxes that apply._

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by oleggrech-t — APPROVED** [review]

File: components/organisms/cards/CardComponent.vue [hunk 1]
```diff
@@ -3,68 +3,80 @@
...
-                                            </b-col>
-                                        </b-row>
-                                        <b-row class="mt-2">
-                                            <b-col>
-                                                <div class="card-number">
-                                                    •••• {{ card.cardNumberLastFour }}
+                <div class="b-aspect d-flex overflow-hidden">
+                    <div
+                        class="b-aspect-sizer flex-grow-1"
+                        style="padding-bottom: 62.5%; height: 0px"
```
Comment (PeterThrels) [inline components/organisms/cards/CardComponent.vue:13]: why these inline styles, why this percentage?

File: nuxt.config.ts [hunk 2]
```diff
@@ -140,4 +141,6 @@ export default defineNuxtConfig({
         strict: true,
         typeCheck: true,
     },
+
+    compatibilityDate: '2024-09-05',
```
Comment (PeterThrels) [inline nuxt.config.ts:145]: is this needed?

File: assets/scss/style.scss [hunk 3]
```diff
@@ -143,3 +143,9 @@ a {
     transition: all 0.2s;
     opacity: 0;
 }
+
+@media (width >= 1400px) {
```
Comment (PeterThrels) [inline assets/scss/style.scss:147]: why is this needed?

Comment (LucasSaid005) [reply assets/scss/style.scss:147]: Bootstrap 5 has an extra media query. So I'm limiting how much the card can expand

File: nuxt.config.ts (hunk 2 above)
Comment (LucasSaid005) [reply nuxt.config.ts:145]: That is automatically generated whenever you run 'pnpm i'

File: components/organisms/cards/CardComponent.vue (hunk 1 above)
Comment (LucasSaid005) [reply components/organisms/cards/CardComponent.vue:13]: had no knowledge about this. will fix it now

Comment (LucasSaid005) [reply components/organisms/cards/CardComponent.vue]: done

**Review by GabCini — APPROVED** [review]

File: components/organisms/cards/CardComponent.vue [hunk 4]
```diff
@@ -3,68 +3,77 @@
...
-                                        </b-row>
-                                        <b-row class="mt-2">
-                                            <b-col>
-                                                <div class="card-number">
-                                                    •••• {{ card.cardNumberLastFour }}
+                <div class="b-aspect d-flex overflow-hidden">
+                    <div class="b-aspect-sizer flex-grow-1"></div>
+                    <div
+                        class="b-aspect-content flex-grow-1 w-100 mw-100 pb-4 h-0"
+                        style="margin-left: -100%"
```
Comment (PeterThrels) [inline components/organisms/cards/CardComponent.vue:14]: you changed to this? -100%
did you test this?

**Review by gaucj-t — CHANGES_REQUESTED** [review]

File: nuxt.config.ts (hunk 2 above)
Comment (PeterThrels) [reply nuxt.config.ts:145]: This was updated because you selected 'yes'

File: assets/scss/style.scss [hunk 5]
```diff
@@ -143,3 +143,9 @@ a {
...
+    .container {
+        max-width: 1128px;
+    }
+}
```
[the remark this answers is not in the document]
Comment (LucasSaid005) [reply assets/scss/style.scss:151]: The card looks bigger than on qa because of the extra media query bootstrap 5 has over 4. Do i leave it as it is or delete it?

File: components/organisms/cards/CardComponent.vue (hunk 4 above)
Comment (LucasSaid005) [reply components/organisms/cards/CardComponent.vue:14]: done

File: assets/scss/_card.scss [hunk 6]
```diff
@@ -1,4 +1,8 @@
 .card {
+    .card-body {
+        padding: 20px;
+    }
+
```
[the remark this answers is not in the document]
Comment (LucasSaid005) [reply assets/scss/_card.scss:5]: Removed it as it is unnecessary
